### PR TITLE
Bschnurr/Microsoft.Python.Parsing nuget publish

### PR DESCRIPTION
**Pull request by bschnurr**
- Bundling Microsoft.Python.Parsing into a nuget package for PTVS
- Removing the old Anaylsis nuget bundle
- Updating copywrite info to pass nuget push command verification
https://aka.ms/Microsoft-NuGet-Compliance
- Updating signing to sign Microsoft.Python.Parsing package instead of analysis

### CODE REVIEW

File: src/Parsing/Impl/Microsoft.Python.Parsing.csproj [hunk 1]
```diff
@@ -35,6 +35,11 @@
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <None Update="Microsoft-Python-Parsing.nuspec">
```
Comment (heejaechang) [inline src/Parsing/Impl/Microsoft.Python.Parsing.csproj]: Microsoft.Python.Parsing.nuspec ?

Comment (MikhailArkhipov) [reply src/Parsing/Impl/Microsoft.Python.Parsing.csproj]: I think nuget does not tolerate dots.

**Comment by heejaechang** [comment]
so replacing analysis to parsing rather than just adding parsing nuget pkg?

**Comment by MikhailArkhipov** [comment]
Engine package included parser since it is a dependency. I guess analysis is no longer needed since VS works via LSP.

File: src/Parsing/Impl/Microsoft.Python.Parsing.csproj (hunk 1 above)
Comment (bschnurr) [reply src/Parsing/Impl/Microsoft.Python.Parsing.csproj]: I just keep the nuspec name the same style as before.. but the nuget id is using ```.```

**Comment by bschnurr** [comment]
> Engine package included parser since it is a dependency. I guess analysis is no longer needed since VS works via LSP.

Correct.. @Huges has duplicated the one or two files he needed from analysis

**Comment by bschnurr** [comment]
> so replacing analysis to parsing rather than just adding parsing nuget pkg?

Yes. PTVS was the only user of analysis but in reality they just need parsing.

**Pull request merged**